flocker/common/_defer.py

```diff
@@ -41,7 +42,7 @@ def log_and_discard(failure):
 
         :param Failure failure: The ``Failure`` to be logged.
         """
-        log.err(failure)
+        write_failure(failure)
 
     for deferred in deferreds:
         deferred.addErrback(log_and_discard)
```

Comment:
:grey_question: no action required, nit: I think you can just use write_failure directly here, we use write_failure to discard failures elsewhere in the code - I understand if you want it explicit, but it's fairly explicitly that way in the API (https://github.com/ClusterHQ/eliot/blob/88698c7c4603065413430edfd19018cbfa0a2ff2/eliot/_traceback.py#L99)